rust/osm2lanes/src/transform.rs

```diff
@@ -502,34 +512,129 @@ fn osm_separation_type(x: &str) -> Option<BufferType> {
 
 pub fn lanes_to_tags(lanes: &[LaneSpec], _cfg: &Config) -> Result<Tags, LaneSpecError> {
     let mut tags = std::collections::BTreeMap::new();
+    let mut oneway = false;
     tags.insert("highway".to_owned(), "yes".to_owned()); // TODO, what?
+                                                         // Lanes
     {
         let lane_count = lanes
             .iter()
-            .filter(|lane| lane.lane_type == LaneType::Driving)
+            .filter(|lane| {
+                lane.lane_type == LaneType::Driving || lane.lane_type == LaneType::SharedLeftTurn
```

Comment:
https://wiki.openstreetmap.org/wiki/Key:lanes#Description
Why not `LaneType::Bus`?

Reply:
TDD, I just didn't think about it

Reply:
- [ ] LaneType::Bus